src/main/java/adapters/login_leaderboard/LeaderboardPresenter.java

```diff
@@ -0,0 +1,5 @@
+package adapters.login_leaderboard;
+
+public class LeaderboardPresenter {
+    // Transforms the information from all three leaderboards into a viewable format.
```

Comment:
Javadoc should be written before the class, in the format of /** **/

Reply:
Also just a side note: commit messages shouldn't be repeated, as it becomes difficult for others to see what you changed in the commit. It doesn't have to be long, just make sure that the commit message describes what you changed from next time!